src/parser.c

```diff
@@ -559,3 +559,23 @@ size_t pos_to_str_with_limit(struct pos_array *pos, uint8_t *str, size_t limit)
     }
     return len;
 }
+
+int pos_is_zero(struct pos_array *pos)
```

Comment:
To make use of `strtoll`, a `pos_array` must be converted into a string, too expensive for an operation like this.